### PR TITLE
translation update

### PR DESCRIPTION
**Pull request by eduardoaddad**
translation update

### CODE REVIEW

File: plugins/UiConfig/languages/pt-br.json [hunk 1]
```diff
@@ -30,4 +30,15 @@
 	"Save settings": "Salvar configurações",
 	"Some changed settings requires restart": "Algumas configurações alteradas requrem reinicialização",
 	"Restart ZeroNet client": "Reiniciar cliente da ZeroNet"
```
Comment (purplesyringa) [inline plugins/UiConfig/languages/pt-br.json]: You missed a comma at the end of this line. Can you add it please?

Comment (eduardoaddad) [reply plugins/UiConfig/languages/pt-br.json]: done

Comment (purplesyringa) [reply plugins/UiConfig/languages/pt-br.json]: You still haven't fixed the issue. There should be a comma (,) at the end of the line.

Comment (eduardoaddad) [reply plugins/UiConfig/languages/pt-br.json]: now I made the correction

**Pull request merged**